lib/RateLimit.js

```diff
@@ -21,9 +21,9 @@ export class RateLimit {
     #attempts = new Map();
     /**
      * Name of the rate limit
-     * @private
+     * @readonly
      */
-    #name;
+    name;
```

Comment:
Maybe you can remove it from here
```suggestion
```

Reply:
This is generated code. I don't want to change it as it will be generated again. Besides, this is the only way to add a jsdoc to this property